reference/hoon-expressions/rune/bar.md

```diff
@@ -8,12 +8,78 @@ Core expressions produce cores. A core is a cell of `[battery payload]`.
 The `battery` is code, a battery of Nock formulas.  The `payload` is the data
 needed to run those formulas correctly.
 
-Four core runes (`|=`, `|.`, `|-`, and `|*`) produce a
+Five core runes (`|=`, `|.`, `|-`, `|*`, and `|$`) produce a
 core with a single arm, named `$`. We can recompute this arm
-with changes, which useful for recursion among other things:
+with changes, which is useful for recursion among other things.
```

Comment:
> We can recompute this arm

This is true of any arm, but this framing implies that might not be the case.

Reply:
https://github.com/urbit/docs/pull/877/commits/0a751e0c212968d4fbd362a9d6713e133fb84195